flow-typed/@sentry/react-native_v1.x.x.js

```diff
@@ -0,0 +1,187 @@
+// flow-typed signature: a24669b78520403133fc3a2153278527
+// flow-typed version: <<STUB>>/@sentry/react-native_v1.0.9/flow_v0.92.0
+
+/**
+ * This is an autogenerated libdef stub for:
```

Comment:
We can delete all the autogenerated goo below, and this autogeneration header with it. The goo is only useful for making things omni-permissive, and with your hand-written types that's no longer needed :slightly_smiling_face: or desired.